### ISSUE TITLE
Cannot disable populate on a per-controller basis

### ISSUE THREAD

**Issue by gotmikhail** [balderdashy/sails#2649]
Using sails v0.11.0

http://sailsjs.org/#!/documentation/reference/blueprint-api?q=blueprint-routes
https://github.com/balderdashy/sails-docs/blob/master/reference/blueprint-api/blueprint-api.md#disabling-blueprints-on-a-per-controller-basis

According to documentation you should be able to disable `populate`, but adding it to the `_config` override in the controller has no effect.

```javascript
module.exports = {
  _config: {
    populate: false
  }
}
```

This is possibly an error in documentation. If the documentation is wrong and `populate` is working as expected, it would be great if `sails` actually did support enabling / disabling `populate` on a granular basis without overriding the blueprint altogether.

**Comment by sailsbot** [comment]
Thanks for posting, @gotmikhail.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
